setup.py

```diff
@@ -67,6 +67,7 @@
     long_description_content_type='text/x-rst',
     project_urls={
         "Code": "https://github.com/sphinx-doc/sphinx",
+        "Changelog": "https://www.sphinx-doc.org/en/master/changes.html",
```

Comment:
For consistency with the name of the file.
```suggestion
        "Changes": "https://www.sphinx-doc.org/en/master/changes.html",
```

Reply:
I would stick with "changelog" to be consistent with the page title and the many other projects that have a "changelog" link in their sidebar (I must have contributed this to like a hundred repos now).